Pipfile

```diff
@@ -5,6 +5,7 @@ name = "pypi"
 
 [packages]
 "discord.py" = {ref = "rewrite", git = "https://github.com/Rapptz/discord.py"}
```

Comment:
The `discord.py` requirement will need to be changed to:

```python
discord-py = {extras = ["voice"], ref = "rewrite", git = "https://github.com/Rapptz/discord.py"}
```

`PyNaCl` is already a requirement for this so it doesn't need to be added to our Pipfile

Reply:
@meooow25 Could you please make that requested change so that your PR is ready to be merged?

Reply:
Apologies for the delay, I am having trouble with locking the pipenv due to incompatible dependencies. I asked about it in the Discord server but received no solution.

`Could not find a version that matches pycodestyle<2.4.0,<2.5.0,<3.0.0,>=2.0.0,>=
2.4.0`

`pycodestyle < 2.4.0` and `>= 2.4.0` is of course impossible to satisfy. Some of the `flake8` packages are probably causing this. What should be done here?

Reply:
This is some other issue with your environment, I just downloaded your branch and it installs & locks just fine.

We'll have to do some more research to figure out what's going on.